doc/design/modes/README.md

```diff
@@ -0,0 +1,25 @@
+# Modes
+
+Mongoose can be launched in 2 main modes:
+* **Standalone** mode
+* **Distributed** mode
+
+## Standalone mode
+
+![Standalone Mode](../../images/standalone_mode.png)
+
+This is the easiest way to use mongoose. In this case, one mongoose instance is launched and the load on the storage generated from it. This mode **does not require any additional parameters**.
```

Comment:
This mode **does not require any additional parameters** - this is a bit unclear, what do you mean by that? Standalone mode might need a lot of parameters depending on a specific storage. I guess what you wanted to say is that it doesn't need additional parameters compared to additional parameters required for distributed mode, right?

Reply:
you are right

Reply:
https://github.com/emc-mongoose/mongoose-base/blob/BASE-1389-documentation/doc/design/modes/README.md#standalone-mode